functions/primeNumber.js

```diff
@@ -0,0 +1,9 @@
+// Напишите функцию, которая определяет является ли число простым или нет
+
```

Comment:
Not fully solved. This function returns `true` for negative numbers, 0 and 1. But it should return `false` for these cases.